src/utils/useCurrency.ts

```diff
@@ -140,16 +140,25 @@ const useCurrency = () => {
       const multiplier = currentPrices?.[currencyType.toLowerCase()] || 0
 
       const showAsHnt = !convert || !multiplier
+
       if (showAsHnt) {
         if (split) {
-          const [intStr, decStr] = balance
+          let [intStr, decStr] = balance
             .toString(undefined, {
               decimalSeparator,
               groupSeparator,
               showTicker: false,
             })
             .split(decimalSeparator)
 
+          // when there is no network the toString method from helium js may not work properly
+          if (intStr === '[object Object]') {
```

Comment:
should we fix helium-js?

Reply:
Its not helium-js its that the object that gets persisted doesnt get the toString method properly so it just ends up returning `[object Object]` when its loaded from cache